### ISSUE TITLE
GRUB booting from UEFI doesn't work for whatever reason.

### ISSUE THREAD

**Issue by Sonicadvance1** [Sonicadvance1/linux#4]
Cross-compiling GRUB is already impossible. So you need an ARMv8 device to compile GRUB for. Also I haven't been able to correctly compile it anyway.
So I've been ripping GRUB from Fedora's AArch64 image to boot.
https://ftp-stud.hs-esslingen.de/pub/fedora-secondary/releases/31/Workstation/aarch64/images/

If you try booting the kernel directly from this GRUB image then it doesn't seem to work very well.
So currently I'm booting the device with a hardcoded command line directly from UEFI instead.

**Comment by klardotsh** [comment]
Have you tried Linaro's Grub fork? https://github.com/aarch64-laptops/grub

I was able to cross-compile that fork when I tried running Void Linux on an aarch64 device (did all the packaging on my desktop and rsync'd the files over to my Yoga C630 WOS): https://github.com/klardotsh/void-packages/commit/aed98d9c5ae5a7bb02a46e3f1a476204206a1288. I've since moved the machine to Gentoo and build on-device anyway.
